source/fundamentals/serialization/guid-serialization.txt

```diff
@@ -24,87 +24,104 @@ In this guide, you can learn how to serialize **globally unique identifiers**
 (`GUIDs <https://learn.microsoft.com/en-us/dynamicsax-2012/developer/guids>`__),
 also known as **universally unique identifiers** (UUIDs).
 
-.. tip::
+A Short History of MongoDB GUIDs
+--------------------------------
 
-   In MongoDB applications, ``ObjectId`` can be used as a unique identifier for
-   a document. Consider using ``ObjectId`` in place of a GUID with MongoDB
-   applications where possible.
+A GUID is a 16-byte integer that you can use as a unique ID for a MongoDB document.
+The following code block shows an example GUID:
 
-A GUID is a 16-byte integer that you can use as a unique ID for a MongoDB document. 
-Originally, GUIDs in MongoDB were represented as ``BsonBinaryData`` values of subtype 3. 
-Subtype 3 did not standardize the byte order during serialization, which led to 
-inconsistent serialization across MongoDB drivers. 
-To standardize the byte order and ensure consistent serialization across drivers, we 
-created ``BsonBinaryData`` subtype 4.
+.. code-block::
 
-.. note::
-   
-   Use ``BsonBinaryData`` subtype 4 for all new GUIDs. 
+  00112233-4455-6677-8899-aabbccddeeff
 
-GuidRepresentationMode
-----------------------
+Originally, MongoDB represented GUIDs as ``BsonBinaryData``
+values of subtype 3. Because subtype 3 didn't standardize the byte order of GUIDs
+during encoding, different MongoDB drivers encoded GUIDs with different byte orders.
 
-In many MongoDB collections, all GUID fields use the same subtype of ``BsonBinaryData``.
-Some older collections, however, may contain some GUID fields that 
-use subtype 3 and others that use subtype 4. 
-To ensure that the driver serializes and deserializes all GUIDs correctly,  
-you should set the ``BsonDefaults.GuidRepresentationMode`` property to one of the 
-following ``GuidRepresentationMode`` values: 
+Use the following tabs to compare the ways in which different MongoDB language drivers
+encoded the preceding GUID to ``BsonBinaryData`` subtype 3:
```

Comment:
```suggestion
The following tabs contain different driver encodings of the preceding GUID to ``BsonBinaryData`` subtype 3:
```

Reply:
Don't mind this change, but can you help me understand why it's preferable?

Reply:
I think it's clearer, less convoluted